xlml/utils/metric.py

```diff
@@ -523,4 +531,7 @@ def process_metrics(
   print("Test run rows:", test_run_rows)
 
   if is_valid_entry():
+    # delete records from BigQuery tables for the same Airflow run_id (if applies),
+    # then, insert new records from current run.
+    bigquery_metric.delete(row_ids)
```

Comment:
QQ, the `row_ids` seems to be different between two airflow run because the `base_id` is different. What are we trying to delete here?

Reply:
I think I update that logic already, and now it should be the same. Please take a look at here: https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/85/files#diff-74b29a75aab611eb1dff01dac1600bc202aaf83cc6376e182e00676aa6ff06ccR349